app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -231,6 +232,13 @@ class InstalledAddonDetailsFragment : Fragment() {
         }
     }
 
+    private fun setAllInteractiveViewsClickable(clickable: Boolean) {
+        enable_switch.isClickable = clickable
+        details.isClickable = clickable
+        permissions.isClickable = clickable
+        remove_add_on.isClickable = clickable
+    }
```

Comment:
Could we use view to call this references, as some of them get execute inside `runIfFragmentIsAttached` we could get a null-pointer exception in some cases

```suggestion
    private fun setAllInteractiveViewsClickable(view: View,clickable: Boolean) {
        view.enable_switch.isClickable = clickable
        view.details.isClickable = clickable
        view.permissions.isClickable = clickable
        view.remove_add_on.isClickable = clickable
    }
```